wagtail/embeds/models.py

```diff
@@ -58,3 +60,18 @@ def is_responsive(self):
 
     def __str__(self):
         return self.url
+
+    def validate_url_max_width_unique(self, error):
+        qs = self.__class__.objects.filter(url=self.url, max_width=self.max_width)
+        if self.id:
+            qs = qs.exclude(id=self.id)
+        if qs.exists():
+            raise error(_('Embed with this Url and Max width already exists.'))
+
+    def clean(self):
+        self.validate_url_max_width_unique(ValidationError)
+        super().clean()
```

Comment:
Might it be worth overriding `validate_unique()` instead of clean() here? https://github.com/django/django/blob/master/django/db/models/base.py#L1009